sycl/include/CL/sycl/detail/pi.h

```diff
@@ -529,10 +529,12 @@ pi_result piMemRetain(
 pi_result piMemRelease(
   pi_mem mem);
 
-pi_mem piMemBufferPartition( // TODO: change interface to return error code
-    pi_mem context, pi_mem_flags flags,
-    pi_buffer_create_type buffer_create_type, void *buffer_create_info,
-    pi_result *errcode_ret);
+pi_result piMemBufferPartition(
+    pi_mem 			buffer,
+    pi_mem_flags 		flags,
+    pi_buffer_create_type 	buffer_create_type,
+    void *			buffer_create_info,
+    pi_mem *			ret_mem);
```

Comment:
Please, replace tabs with spaces.
If you are not going to apply clang-format, you can just search for tabs.

Reply:
done.